src/pages/User/Settings/Learning/Learning.tsx

```diff
@@ -0,0 +1,80 @@
+import { useForm, useFieldArray, Controller } from 'react-hook-form';
+
+import TimePickerViewRenderers from 'components/Shared/Form/Time';
+
+import * as S from './StyleLearning';
+import Submit from 'components/Shared/Form/Submit';
+import ManagedIcon from 'assets/icon/helpers/ManagedIcon';
+import Select from 'components/Shared/Form/Select';
+import { ButtonIcon } from 'components/Shared/Buttons/Button';
+import { TitleSmall, TitleBig } from 'components/Shared/Atoms/Title';
+import useLearning from './useLearning';
+
+const LearningSettings: React.FC = () => {
+  const {
+    handleRemoveNotification,
+    handleAddNotification,
+    isAddButtonDisabled,
+    daysOfWeek,
+    watch,
+    handleSubmit,
+    setValue,
+    onSubmit,
+    fields,
+    control,
+  } = useLearning();
+
```

Comment:
skoro komponent to LerningSettings, to nazwa pliku i folderu też powinna taka być. to samo dotyczy `useLearningSettings`. żeby to wszystko spójne było